src/Core/Services/RestService.cs

```diff
@@ -334,7 +334,7 @@ private bool TryGetStoredProcedureRESTVerbs(string entityName, [NotNullWhen(true
             {
                 if (runtimeConfig.Entities.TryGetValue(entityName, out Entity? entity))
                 {
-                    SupportedHttpVerb[] methods = entity.Rest.Methods;
+                    SupportedHttpVerb[] methods = (entity.Rest.Methods is not null) ? entity.Rest.Methods : Array.Empty<SupportedHttpVerb>();
```

Comment:
Shouldn't the methods array here be populated with POST - the default method for SP as per the function comment?

Reply:
Thanks for the suggestion. Updated the logic to populate withPOST when the REST APIs are enabled for the entity